lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -350,6 +354,37 @@ func (h *StorageHandlersImpl) WriteImage(params storage.WriteImageParams) middle
 	return storage.NewWriteImageCreated().WithPayload(i)
 }
 
+//ImageJoin modifies the config spec of a container to include the specified image
+func (h *StorageHandlersImpl) ImageJoin(params storage.ImageJoinParams) middleware.Responder {
+	op := trace.NewOperation(context.Background(), "ImageJoin %s", params.ID)
+	defer trace.End(trace.Begin("", op))
+
+	handle := epl.HandleFromInterface(params.Config.Handle)
+	if handle == nil {
+		err := &models.Error{Message: "Failed to get the Handle"}
```

Comment:
Maybe better to phrase it like "Handle not found"?

Reply:
I've left it consistent with the `task.Join` call (task_handles.go:53) - not going to debate that the message could be better, but trying to limit the scope of the change.